docs/gl_objects/invitations.rst

```diff
@@ -0,0 +1,73 @@
+###########
+Invitations
+###########
+
+Invitations let you invite or add users to a group or project.
+
+Reference
+---------
+
+* v4 API:
+
+  + :class:`gitlab.v4.objects.GroupInvitation`
+  + :class:`gitlab.v4.objects.GroupInvitationManager`
+  + :attr:`gitlab.v4.objects.Group.invitations`
+  + :class:`gitlab.v4.objects.ProjectInvitation`
+  + :class:`gitlab.v4.objects.ProjectInvitationManager`
+  + :attr:`gitlab.v4.objects.Project.invitations`
+
+* GitLab API: https://docs.gitlab.com/ce/api/invitations.html
```

Comment:
Shouldn't this be: https://docs.gitlab.com/ee/api/invitations.html  ?

s/ce/ee/

Reply:
This is available in all tiers, it's just that GitLab is pushing for EE now so it redirects /ce/ to /ee/ by default, so we probably get more of these recently but otherwise the same I think. We still have more /ce/ links in our docs IIRC.

Reply:
Okay. But they are marking it as **permanently** moved

```
$ wget 'https://docs.gitlab.com/ce/api/invitations.html'
--2022-07-23 14:30:24--  https://docs.gitlab.com/ce/api/invitations.html
Resolving docs.gitlab.com (docs.gitlab.com)... 35.185.44.232
Connecting to docs.gitlab.com (docs.gitlab.com)|35.185.44.232|:443... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: /ee/api/invitations.html [following]
--2022-07-23 14:30:25--  https://docs.gitlab.com/ee/api/invitations.html
```